src/components/select/select.component.spec.ts

```diff
@@ -43,6 +43,10 @@ describe('Select Component', () => {
         fixture.detectChanges();
     });
 
+    it('should be initially blank if no placeholder is set', () => {
+        expect(component.input).toBe('');
+    });
+
```

Comment:
Would it be possible to add tests to cover an initial value when using `value`, when using `ngModel` and when using `ReactiveForms`? Just so this issue won't regress again.

And possibly a test to ensure the value doesn't immediate get emitted when an initial value is set in each case?